test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -202,8 +202,8 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.graph.savePolicy (alternatives)',
         'xpack.ilm.ui.enabled (boolean)',
         'xpack.index_management.ui.enabled (boolean)',
-        'xpack.infra.logs.app_target (alternatives)',
         'xpack.infra.sources.default.fields.message (array)',
+        'xpack.infra.logs (any)',
```

Comment:
Is the reason this is set to `any` instead of `string` because it is optional?

If it can't be changed to `string` please add an inline comment that describes what type of information is expected (expected values if possible)

Reply:
> Is the reason this is set to any instead of string because it is optional?

Yes it is, this will be optionally passed, I just added a comment in the code, thanks for checking!